sigstore-java/build.gradle.kts

```diff
@@ -67,7 +63,7 @@ protobuf {
     generateProtoTasks {
         ofSourceSet("main").forEach() {
             it.plugins {
-                id("grpc")
+                id("grpc") {}
             }
         }
```

Comment:
Please use the following here:

```kotlin
    generateProtoTasks {
        ofSourceSet("main").configureEach {
            plugins {
                id("grpc")
            }
        }
    }
```

Reply:
Oops must've left that in there while experimenting

Reply:
I would suggest replacing `forEach()` with `configureEach`, so `it.` is not needed before `plugins`

Reply:
Damn, I missed that whole suggestion

Reply:
I logged an issue for streamlining protobuf-gradle-plugin configuration: https://github.com/google/protobuf-gradle-plugin/issues/628